packages/jest-config/src/normalize.ts

```diff
@@ -176,11 +183,17 @@ const setupPreset = (
       );
     }
 
-    throw createConfigError(
-      `  An unknown error occurred in ${chalk.bold(presetPath)}:\n\n  ${
-        error.message
-      }\n  ${error.stack}`,
-    );
+    if (presetModule && path.extname(presetModule) === '.mjs') {
+      throw createConfigError(
```

Comment:
I wonder if we need this specific error path or we can use the generic path (in the `else` block)